src/main/java/meditracker/MediTracker.java

```diff
@@ -77,7 +78,7 @@ public void run() {
 
             try {
                 command.execute();
-            } catch (InvalidArgumentException e) {
+            } catch (InvalidArgumentException | MediTrackerException e) {
```

Comment:
Id suggest @wenenhoe to review as he may have handled a similar exception in his previous PR, as discussed in our recent group meeting.